proto/p4/v1/p4runtime.proto

```diff
@@ -264,6 +265,16 @@ message Action {
   repeated Param params = 4;
 }
 
+message ActionProfileActions {
+  repeated ActionProfileAction action_profile_action = 1;
```

Comment:
Per protobuf style guide, repeated field names should be plural.

Reply:
Done.